### PR TITLE
Add advisory for SmallVec issue #148

### PR DESCRIPTION
**Pull request by Shnatsel**
https://github.com/servo/rust-smallvec/issues/148

### CODE REVIEW

File: crates/smallvec/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+
+# Name of the affected crate (mandatory)
```
Comment (tarcieri) [inline crates/smallvec/RUSTSEC-0000-0000.toml]: Please remove the comments

**Comment by lucab** [comment]
Was a CVE already requested/assigned for this? If not, is anybody planning to request one? I'd be interesting in that and can offer some help if needed.

**Comment by Shnatsel** [comment]
Help with a CVE would be great! Last time I tried to get a CVE through iwantacve.org I've waited for months and never heard back. It's been https://github.com/rust-lang/rust/issues/53566 and I still want one, by the way.

Do you think it's also a good idea to include https://github.com/servo/rust-smallvec/issues/149 in the same CVE? It has a very similar exploitation pattern. I've failed to determine the exact point where the bug was introduced, though.

**Comment by lucab** [comment]
@Shnatsel I'd suggest to have an advisory for servo/rust-smallvec#149 too (the fixed-version is enough, as a start) and wait for @tarcieri to assign stable IDs for both.
I can then try to forward this to my company secteam for CVE bureaucracy, if nobody gets to that in a quicker way.

**Comment by attritionorg** [comment]
This was just assigned CVE-2019-1010299.

**Comment by Shnatsel** [comment]
@lucab the other smallvec issue is in as RUSTSEC-2019-0012, so please proceed with the CVE magic.

**Comment by lucab** [comment]
I think @attritionorg has a quicker channel than me, so I'll wait for their feedback first in order to avoid getting duplicated IDs.

**Comment by attritionorg** [comment]
@lucab I do not. MITRE has an unofficial policy to ignore any mail from me, on any topic, including CVE questions/disputes/assignments.

**Comment by Shnatsel** [comment]
The assignment of CVE-2019-1010299 was my application via iwantacve.org from February finally coming through.

**Pull request merged**